### PR TITLE
[Domain-Seeds] Adding CIS Domain seeds

### PR DESCRIPTION
**Pull request by rajivmucheli**
Hi @Carthaca,

Please validate the PR.

Regards,
Rajiv

### CODE REVIEW

**Review by BerndKue — APPROVED** [review]

File: openstack/barbican/templates/seed.yaml [hunk 1]
```diff
@@ -509,3 +510,41 @@ spec:
...
       #  role: keymanager_viewer
       #  inherited: true
+
+  - name: cis
+    groups:
+    - name: CIS_API_SUPPORT
+      role_assignments:
+      - project: api_support
+        role: keymanager_admin
+      - domain: neo
```
Comment (Carthaca) [inline openstack/barbican/templates/seed.yaml:520]: `cis`

File: openstack/barbican/templates/seed.yaml [hunk 2]
```diff
@@ -509,3 +510,41 @@ spec:
...
+        role: keymanager_viewer
+        inherited: true
+    - name: CIS_COMPUTE_SUPPORT
+      role_assignments:
+      - project: compute_support
+        role: keymanager_admin
+      #- domain: neo
```
Comment (Carthaca) [inline openstack/barbican/templates/seed.yaml:527]: either remove the comment of fix the domain name here, too, please

File: openstack/designate/templates/seeds.yaml [hunk 3]
```diff
@@ -29,6 +29,7 @@ spec:
   - {{ .Release.Namespace }}/domain-neo-seed
   - {{ .Release.Namespace }}/domain-s4-seed
   - {{ .Release.Namespace }}/domain-wbs-seed
+  - {{ .Release.Namespace }}/monsoon3/domain-cis-seed
```
Comment (Carthaca) [inline openstack/designate/templates/seeds.yaml:32]: `{{ .Release.Namespace }}/domain-cis-seed`

"monsoon3" is the Namespace, would be duplicated

File: openstack/designate/templates/seeds.yaml [hunk 4]
```diff
@@ -710,6 +711,54 @@ spec:
...
+      - domain: cis
+        role: cloud_dns_ops
+        inherited: true
+      - domain: cis
+        role: role_viewer
+        inherited: true
+      - domain: cis
+        role: audit_viewer
+        inherited: true
+    - name: NEO_API_SUPPORT
```
Comment (Carthaca) [inline openstack/designate/templates/seeds.yaml:732]: `CIS_API_SUPPORT`

File: openstack/domain-seeds/templates/domain-cis-seed.yaml [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  requires:
+  - {{.Values.global.keystoneNamespace}}/keystone-seed
+
+  domains:
+  - name: cis
+    description: CIS Platform
```
Comment (Carthaca) [inline openstack/domain-seeds/templates/domain-cis-seed.yaml:16]: I think this is the abbrev. of `Central Infrastructure Services`

File: openstack/keppel/templates/support_seed.yaml [hunk 6]
```diff
@@ -1,4 +1,4 @@
-{{- $domains := list "ccadmin" "bs" "cp" "fsn" "hcm" "hcp03" "hec" "monsoon3" "neo" "s4" "wbs"}}
+{{- $domains := list "ccadmin" "bs" "cp" "cis" "fsn" "hcm" "hcp03" "hec" "monsoon3" "neo" "s4" "wbs"}}
```
Comment (Carthaca) [inline openstack/keppel/templates/support_seed.yaml:1]: please put it in alphabetical order between `bs` and `cp`

**Review by Carthaca — CHANGES_REQUESTED** [review]
mostly copy&paste errors

**Review by Carthaca — APPROVED** [review]
lgtm, the ci failures seem to be unrelated

**Pull request merged**